wallet/src/de/schildbach/wallet/ui/InputParser.java

```diff
@@ -452,7 +452,8 @@ protected void dialog(final Context context, @Nullable final OnClickListener dis
 
 	private static final Pattern PATTERN_BITCOIN_ADDRESS = Pattern.compile("[" + new String(Base58.ALPHABET) + "]{20,40}");
 	private static final Pattern PATTERN_DUMPED_PRIVATE_KEY_UNCOMPRESSED = Pattern.compile((Constants.NETWORK_PARAMETERS.getId().equals(
```

Comment:
Hm, why not move the entire pattern to Constants? I think some altcoins have multiple possible digits so a single char won't do.